nekoyume/Assets/_Scripts/UI/Widget/QuestPreparation.cs

```diff
@@ -500,6 +500,8 @@ private void ReadyToQuest(bool ready)
                     particle.Stop();
                 }
             }
+
+            questButton.SetConditionalState(ready);
```

Comment:
`ConditionalCostButton` already has a feature that checks current avatar has enough cost.
I recommend you to remove `ready` parameter in `ReadyToQuest` and make button update itself with `questButton.UpdateObjects`.

Reply:
Thanks for your suggestion. I'll try it!